### ISSUE TITLE
Contact page has errors

### ISSUE THREAD

**Issue by vegasbrianc** [flavioaiello/flavioaiello.github.io#1]
Very cool template. I noticed the contact page is missing an image in the header and the footer is not loading correctly.

**Comment by flavioaiello** [comment]
Yes it needs time (I didn't had yet) to customize ... github pages with jekyll is cool, there are very nice themes out there. search per example for «html5up themes jekyll» ... they look awesome
